editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -2040,6 +2043,8 @@ TileMapEditor::TileMapEditor(EditorNode *p_editor) {
 
 	// Tile position
 	tile_info = memnew(Label);
+	tile_info->set_align(Label::Align::ALIGN_RIGHT);
+	tile_info->set_custom_minimum_size(Size2(200, 0));
```

Comment:
The size should be multiplied by `EDSCALE` for hiDPI support
```suggestion
	tile_info->set_custom_minimum_size(Size2(200, 0) * EDSCALE);
```